compiler/smir/README.md

```diff
@@ -0,0 +1,112 @@
+This crate is regularly synced with its mirror in the rustc repo at `compiler/rustc_smir`.
```

Comment:
I'm assuming we are only going to mirror `compiler/stable_mir` from now on... is that the case?

Reply:
Deduplicated the readme